depends/hosts/darwin.mk

```diff
@@ -1,7 +1,7 @@
 OSX_MIN_VERSION=10.14
-OSX_SDK_VERSION=10.15.1
-XCODE_VERSION=11.3.1
-XCODE_BUILD_ID=11C505
+OSX_SDK_VERSION=10.15.6
+XCODE_VERSION=12.1
```

Comment:
Do we want to leapfrog to Xcode 12.5 or just leave this? There is a new SDK version 10.15.7 which was a security update, but I don't know if it's relevant to the things we use.

Reply:
Going to leave this as-is for now. May be revisited soon in regards to proper Apple ARM support.